crates/install-wheel-rs/src/wheel.rs

```diff
@@ -131,6 +131,7 @@ fn copy_and_hash(reader: &mut impl Read, writer: &mut impl Write) -> io::Result<
 fn format_shebang(executable: impl AsRef<Path>, os_name: &str) -> String {
     // Convert the executable to a simplified path.
     let executable = executable.as_ref().simplified_display().to_string();
+    let is_relocatable = !executable.contains(['\\', '/']);
```

Comment:
Does this assume that the relative path doesn't contain multiple segments? (That's true for virtual environments, but could be false in general, right?)

Reply:
On second thought, this check is probably unnecessary and I might just pass the already-known `is_relocatable` as an argument, rather than trying to reverse-engineer it.

Reply:
Refactored.